rubocop.yml

```diff
@@ -90,6 +95,8 @@ Style/MutableConstant:
   Enabled: false
 Style/NumericPredicate:
   Enabled: false
+Style/OptionHash:
```

Comment:
Which is exactly why rubocop would flag them, both sound like good candidates to redo. Also it's not that bad: after https://github.com/bbatsov/rubocop/pull/2114 it only flags some names for the option hash, so you'd easily avoid it by renaming `options` here to something more specific, e.g `attributes`.

Rubocop is not your enemy :)